src/components/SearchBar.js

```diff
@@ -283,6 +288,18 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
     }
   }
 
+  //eslint-disable-next-line
+  shareSearch() {
+    const username = AuthUtils.getUserName(AuthUtils.getSavedUser());
+    const yourMessage =
+      'Hey,\n\nI think you would be interested in these search results that I found using the most cognitive and ' +
+      `intuitive search platform called Attivio. Here is the link :)\n${window.location.href}
+      \nLet me know if you have any questions!\n\nBest,\n${username}`;
+    const subject = 'Search results I found using Attivio!';
+    const emailLink = 'email@example.com';
```

Comment:
The target email should be a property too, although it should default to being empty, I think... In general, it doesn't make sense to have the address come up pre-filled to something no one will use... (for a PoV, sure...)